docs/internal/internal-search.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Internal Search
+---
+
+# Internal Search
+
+For internal views that need to take advantage of searching and filtering, we've
+chosen to use [Ransack][ransack].
+
+Ransack is a Ruby gem that makes searching relatively painless. It has excellent
+documentation, but if you're looking for an example of how it's being used on
+DEV, we've implemented it to help searching and sorting user reports.
+
+The view responsible for managing user reports can be found at
+`localhost:3000/internal/reports` and Ransack can be seen in use on the index
+action of the [`internal/feedback_messages_controller`][feedback_messages].
+
+Generally, Ransack should not be used for anything except internal views.
```

Comment:
I would be stricter on this. Let's be explicit saying that ransack has to be limited to internal views. If we ever change our minds we can update the docs

Reply:
Good call. Done!